.github/workflows/builder.yml

```diff
@@ -227,6 +229,7 @@ jobs:
           file: ./Dockerfile
           platforms: ${{ steps.vars.outputs.platform }}
           push: true
+          provenance: false
```

Comment:
Done with cosign in the step below

Reply:
Not sure what you mean with "Done with cosign". Provenance contains information on how containers got built exactly, like time stamps, git hashes and I think also the Dockerfile itself.

It gets included when the container build driver is used. It seems `setup-buildx-action` uses the container build driver by default (https://github.com/docker/build-push-action?tab=readme-ov-file#usage), which means a minimal attestation is created by default (see https://docs.docker.com/build/metadata/attestations/#creating-attestations).

I'd leave it enabled.